javascript/javascript1/week2/candy.js

```diff
@@ -0,0 +1,46 @@
+const boughtCandyPrices = [];
+const boughtCandy = [];
+var amountToSpend = Math.random() * 100;
+function addcandy (candyType,weight) {
+if (candyType == 'Sweet') {
+    boughtCandyPrices.push(weight*0.5);
+    boughtCandy.push()
+}
+else if (candyType == 'Chocolate') {
+    boughtCandyPrices.push(weight * 0.7);
+}
+else if (candyType == 'Toffee') {
+    boughtCandyPrices.push(weight * 1.1);
+} 
+else {
+    boughtCandyPrices.push(weight * 0.03);
+}
```

Comment:
You can use a switch case for this. 
Switch cases are useful when you wanna check a bunch of cases for a variable:
```js
switch(candyType) {
  case "Sweet":
    boughtCandyPrices.push(weight*0.5);
    boughtCandy.push()
    break;
  case "Chocolate":
    boughtCandyPrices.push(weight * 0.7);
    break;
    ......
  default:
    console.log("Candytype not recognized");
    break;
}
```

Reply:
It seems like you're only pushing the candy if it's "sweet". Is that on purpose?